sdc/makefile.common

```diff
@@ -1,27 +1,33 @@
 # Common definitions
 
-SDC_SRC = \
+LIBSDC_SRC = \
 	$(SDC_ROOT)/src/sdc/*.d \
 	$(SDC_ROOT)/src/util/*.d
 
+SDC_DRIVER_SRC = $(SDC_ROOT)/src/driver/*.d
+
 SDC = bin/sdc
 
-ALL_TARGET ?= $(SDC)
+LIBSDC = lib/libsdc.a
+
+ALL_TARGET ?= $(LIBSDC) $(SDC)
 
 LIBD_LLVM_ROOT ?= $(SDC_ROOT)/../libd-llvm
 ALL_TARGET ?= $(SDC) bin/sdc.conf
 
 include $(LIBD_LLVM_ROOT)/makefile.common
 
-SDC_IMPORTS = $(LIBD_LLVM_IMPORTS) -I$(LIBD_LLVM_ROOT)/src
+SDC_IMPORTS = -I$(SDC_ROOT)/src $(LIBD_LLVM_IMPORTS) -I$(LIBD_LLVM_ROOT)/src
 
-$(SDC): obj/sdc.o $(LIBD) $(LIBD_LLVM)
+$(SDC): $(LIBSDC) $(LIBD) $(LIBD_LLVM)
 	@mkdir -p bin
-	gcc -o $(SDC) obj/sdc.o $(ARCHFLAG) $(LDFLAGS)
+	$(DMD) -c -ofobj/driver.o $(SDC_DRIVER_SRC) $(DFLAGS) $(SDC_IMPORTS)
```

Comment:
What do you mean with specific target for this ?
the driver needs to be rebuild if anything changes...